src/HomeScreen.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, {Component} from 'react';
+import SetWithCardList from './secondaryComponents/SetWithCardList';
+import {connect} from 'react-redux';
 
-let HomeScreen = () => 
-    <div>
-        <p>Home Screen</p>
+let mapStateToProps = (state) => {
+    return {setList: state.setList, cardList: state.cardList}
+};
+
+class HomeScreenComponent extends Component {
+    
+    render(){
+    let {setList} = this.props;
+    return (
+    <div className="homeScreen">
```

Comment:
I would pop this out into a dumb component/JSX, then wire that to the connect from redux without making a class.  All the props should be coming from the store.  No need to map it to a new class.

Reply:
Redux will definitely handle storing state for the sets and individual cards; however I do need the class based component to handle the state of the dropdown menu.